packages/data-context/src/sources/migration/legacyOptions.ts

```diff
@@ -86,6 +86,10 @@ const resolvedOptions: Array<ResolvedConfigOption> = [
     name: 'exit',
     defaultValue: true,
     canUpdateDuringTestTime: false,
+  }, {
+    name: 'experimentalCspAllowList',
+    defaultValue: false,
+    canUpdateDuringTestTime: false,
```

Comment:
this isn't a legacy config option so we dont want to add this here.

Reply:
Will remove.